ru/project/toc.md

```diff
@@ -45,6 +45,14 @@ items:
   when: version == 12
 ```
 
+## Постановки и условные операторы {#subtitudes}
+
+Название документа поддерживает [постановки](../syntax/vars#subtitudes) и [условные операторы](../syntax/vars#conditions).
```

Comment:
Doesn't description support substitutions?

Reply:
for toc only title